### ISSUE TITLE
Any way to prevent it from popping to night mode?

### ISSUE THREAD

**Issue by bc** [xz/new.css#28]
thanks! theme is otherwise perfect.

**Comment by 3x** [comment]
Custom themes for night mode will be added in 1.3.

Right now, all the night theme colors are hard-coded overrides of the `:root` variables, which was an oversight on my part. 1.3 will allow custom variables to be set for this, such as `--nc-dark-bg-1`.

**Comment by Glench** [comment]
yeah right now the dark mode theme seems like the default but wasn't when I visited the site a few days ago. Did this change or is my system somehow saying it's in dark mode all the time?

**Comment by bc** [comment]
On my mac running Catalina, it immediately follows the System Preference of dark mode/light mode without refreshing the page.
![image](https://user-images.githubusercontent.com/13772726/83037583-d273e700-9ff0-11ea-96fd-5f45bb1b5898.png)

**Comment by Glench** [comment]
Looks like my 'auto' appearance got screwed up. It shows the right colors when each theme is chosen. I wonder if maybe there could be an automatic fade when going into night mode
